cmd/cli/kubectl-kyverno/exception/load.go

```diff
@@ -63,3 +64,18 @@ func load(content []byte) ([]*kyvernov2beta1.PolicyException, error) {
 	}
 	return exceptions, nil
 }
+
+func SelectFrom(resources []*unstructured.Unstructured) []*kyvernov2beta1.PolicyException {
```

Comment:
This function isn't called anywhere. Did you remove it by mistake or I'm missing something?